app/views/events/new.html.erb

```diff
@@ -0,0 +1,21 @@
+<h1>Create Event</h1>
+<%= form_for @event do |f| %>
+  <%= f.label :description %>
+  <br>
+  <%= f.text_area :description, placeholder: "Describe this event" %>
+  <br>
+  <%= f.label :date %>
+  <br>
+  <%= f.date_field :date %>
+
+  <h3>Select Attendees:</h3>
+  <% @users.each do |user| %>
+    <%= f.check_box :attendees,{multiple: true}, user, false %>
+    <%= f.label user, for: "event_attendees_#{user}" %>
```

Comment:
If I submit an email as a name, user will be `.com`:

![image](https://user-images.githubusercontent.com/22228397/65054706-9d78a600-d933-11e9-86b0-b7597a3289d2.png)

It was under my understanding that the intention was to use `email` field and not `name`, but I don't think this is a deal breaker.

[`OPTIONAL`]